packages/fuels-core/src/code_gen/abigen.rs

```diff
@@ -714,64 +714,64 @@ mod tests {
     #[test]
     fn test_get_custom_types_nested_structs_and_enums() {
         let contract = r#"
-[
-  {
-    "type": "function",
-    "inputs": [
-      {
-        "name": "c",
-        "type": "struct Cocktail",
-        "components": [
+        [
```

Comment:
Is there any reason for the name changes? If they serve a certain purpose (for instance the names must be capitalized or wtv) it's good to write those requirements above in a comment.

Reply:
Nothing changed here except the indent. I do now know why Github is highlighting it like that.